### ISSUE TITLE
optional parameter for network device to bind

### ISSUE THREAD

**Issue by seanjensengrey** [basho-labs/riak-mesos-executor#24]
When executing coverage api queries, Riak reports the ip address it is bound to. If it is set to '0.0.0.0' subsequent calls to retrieve data from that node fail because 0.0.0.0 is not routable. Riak should probably respond with actual routable ip addresses, but in the short term, we would like to pass a parameter into the executor, so it can decide which ip address to bind to.

` "network_interface_name" : "eth0" `

This optional parameter would be a child of "executor" in the framework.json config file. If the parameter is absent, the riak node would continue to bind to `0.0.0.0`

```
"executor": {
	"network_interface_name" : "eth0",
	"url": "https://basholabs.artifactoryonline.com/basholabs/riak-mesos-framework/ubuntu-14.04/executor/1.2.0-0.28.1/riak_mesos_executor-1.2.0-mesos-0.28.1-ubuntu-14.04.tar.gz",
	"package": "riak_mesos_executor-1.2.0-mesos-0.28.1-ubuntu-14.04.tar.gz",
	"cpus": 0.1,
	"mem": 512.0
},

```

**Comment by sanmiguel** [comment]
Strictly I think this should go under the `node` object in the config, as it applies to the riak node, not the executor itself.

**Comment by seanjensengrey** [comment]
agreed

```
        "node": {
            "network_interface_name" : "eth0",
            "url": "...",
            "package": "riak_ts-1.3.1.tgz",
            "explorer-url": "...",
            "explorer-package": "riak_explorer-1.1.1.patch-ubuntu-14.04.tar.gz",
            "patches-url": "...",
            "patches-package": "riak_erlpmd_patches-1.2.0-mesos-0.28.1-ubuntu-14.04.tar.gz",
            "cpus": 1.0,
            "mem": 4096.0,
            "disk": 20000.0
        },
```

**Comment by sanmiguel** [comment]
Closed by #26 , basho-labs/riak-mesos-scheduler#71 and basho-labs/riak-mesos-tools#29
